bottlerocket-agents/src/sonobuoy.rs

```diff
@@ -74,14 +74,30 @@ pub async fn run_sonobuoy(
     ensure!(status.success(), error::SonobuoyRunSnafu);
 
     info!("Running sonobuoy retrieve");
+    let results_filepath = results_dir.join(SONOBUOY_RESULTS_FILENAME);
     let status = Command::new("/usr/bin/sonobuoy")
-        .current_dir(results_dir.to_str().context(error::ResultsLocationSnafu)?)
         .args(kubeconfig_arg.to_owned())
         .arg("retrieve")
+        .arg("--filename")
+        .arg(results_filepath.as_os_str())
         .status()
         .context(error::SonobuoyProcessSnafu)?;
     ensure!(status.success(), error::SonobuoyRunSnafu);
 
+    info!("Sonobuoy testing has completed, printing results");
```

Comment:
Where are we printing the results?

Reply:
It goes to the logs (stdout) because I am using Cmd.status() and not Cmd.output()